src/utilities.cc

```diff
@@ -284,10 +284,8 @@ pid_t GetTID() {
 
 const char* const_basename(const char* filepath) {
   const char* base = strrchr(filepath, '/');
-#ifdef OS_WINDOWS  // Look for either path separator in Windows
   if (!base)
```

Comment:
Backslashes are very unlikely on systems other than Windows. One should better fix the `OS_WINDOWS` define or directly check for `__MINGW__`.